src/WarcraftLegacies.Source/Quests/Sunfury/QuestUpperNetherstorm.cs

```diff
@@ -34,15 +34,15 @@ public sealed class QuestUpperNetherstorm : QuestData
       AddObjective(
         new ObjectiveControlPoint(
           ControlPointManager.Instance.GetFromUnitType(Constants.UNIT_N0CW_FARAHLON_10GOLD_MIN)));
-      AddObjective(new ObjectiveExpire(1430, Title));
+      AddObjective(new ObjectiveExpire(600, Title));
       AddObjective(new ObjectiveSelfExists());
       _rescueUnits = rescueRect.PrepareUnitsForRescue(RescuePreparationMode.HideAll);
     }
 
     /// <inheritdoc/>
     protected override void OnComplete(Faction whichFaction)
     {
-      whichFaction.Player?.AddGold(400);
+      whichFaction.Player?.AddGold(200);
```

Comment:
```suggestion
      whichFaction.Player?.AddGold(GoldReward);
```
You've already got a variable containing the gold reward configuration, might as well use it. 
It's a very good idea to do it that way BTW. I would use this for all gold reward quests.